### PR TITLE
GH-892: ROC The Story Fixes

### PR DESCRIPTION
**Pull request by yuenyauu**
- Adjusted header positioning to be synced with other pages
- Adjusted positioning of carousel arrows for larger phones

### CODE REVIEW

File: ui/src/components/ripple-out/RippleOutStoryContainer.js [hunk 1]
```diff
@@ -135,22 +143,26 @@ const RippleOutStoryContainer = () => {
     <>
       <RippleOutHeroSection />
       <Flex
-        minH="100%"
         flexDir="column"
-        bgGradient="linear(120deg, rgba(132,225,255,0.5), rgba(197,230,212,0.5), rgba(255,235,174,0.5));"
+        background="linear-gradient(180deg, #F0F5FF 10.74%, #E9F6FF 22.35%, #FFFAEC 99.87%)"
```
Comment (ksuhartono97) [inline ui/src/components/ripple-out/RippleOutStoryContainer.js:147]: this change seems to make the gradient very faint and not aligned to the design

image

**Review by tamarayustian — APPROVED** [review]

**Review by ksuhartono97 — CHANGES_REQUESTED** [review]

File: ui/src/components/ripple-out/RippleOutStoryContainer.js (hunk 1 above)
Comment (yuenyauu) [reply ui/src/components/ripple-out/RippleOutStoryContainer.js:147]: oh, should i revert it back to the original? i changed it to standardize the background across the pages so...

Comment (tamarayustian) [reply ui/src/components/ripple-out/RippleOutStoryContainer.js:147]: technically, you are correct already and following the design. the other pages should also follow their own individual designs for the pages. so you can revert back to original

Comment (yuenyauu) [reply ui/src/components/ripple-out/RippleOutStoryContainer.js:147]: okk, noted.

**Review by ksuhartono97 — APPROVED** [review]

**Pull request merged**